src/freenas/usr/local/libexec/freenas-debug/system/system.sh

```diff
@@ -148,7 +148,7 @@ system_func()
 	section_footer
 
 	section_header "Middleware Jobs - 'midclt call core.get_jobs'"
-	midclt call core.get_jobs '[["state", "!=", "SUCCESS"]]' | jq .
+	midclt call core.get_jobs '[["state", "!=", "SUCCESS"]]' '{"extra": {"normalize_result": true}}' | jq .
```

Comment:
What does `normalize` here mean? How is `"********"` instead of an e.g. password "normal"?

Can we use a clearer name please?